app/controllers/orders_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
       dish = Dish.find(dish_id)
       if dish_value.to_i > 0
         if @order.add(dish, dish.price, dish_value.to_i)
-          flash[:notice] = "#{dish.name} was successfully added to order!"
+          #flash[:notice] = "#{dish.name} was successfully added to order!"
```

Comment:
If you remove this line then the entire if statement makes no sense. Or does it? Perhaps use a modifier:
```
flash[:alert] = 'Item not added, try again!' unless @order.add(dish, dish.price, dish_value.to_i)
```
but I'm not sure if I like it...